contrib/digitalocean/README.md

```diff
@@ -1,3 +1,23 @@
 # Provision a Deis Cluster on DigitalOcean
 
-Please refer to the instructions at http://docs.deis.io/en/latest/installing_deis/digitalocean/.
+Using Rigger:
+
+    ./rigger configure
+      # answer "Digital Ocean when asked what provider you want to use"
+      # you'll also need to provide your Digital Ocean token and SSH fingerprint
+
+    ./rigger provision
+      # use your new Deis cluster when this command completes!
+
+    ...
+
+    ./rigger destroy
+      # when done with the cluster
+
+Using Terraform:
+
+    terraform apply
+      # will ask you some questions to fill in variables
+      # use your new Deis cluster when this command completes!
+
+Then follow the instructions on [provisioning Deis on Digital Ocean manually](http://docs.deis.io/en/latest/installing_deis/digitalocean/)
```

Comment:
All for it. I wrote this README months back when `rigger` wasn't as blessed as a thing yet.